.github/workflows/benchmarking.yml

```diff
@@ -11,7 +11,7 @@ on:
 
       # test changes to benchmarking implementation
       - "Samples/iOS-Swift/**"
-      - "Samples/iOS-Swift/iOS-Benchmarking/**"
+      - "Samples/iOS-Benchmarking/**"
```

Comment:
`h`: This looks wrong to me, if I checkout the branch and run `make xcode`, the folder is at `iOS-Swift/iOS-Benchmarking`

Reply:
hmm, you're right, but anyways this be redundant with line 13, so i'm actually just going to remove line 14 altogether.